google/cloud/storage/tests/run_integration_tests.sh

```diff
@@ -56,20 +58,30 @@ function start_httpbin {
   done
 
   if [ "${connected}" = "no" ]; then
-    echo "Cannot connect to httpbin; aborting test." >&2
+    echo "Cannot connect to testbench; aborting test." >&2
     exit 1
   else
-    echo "Successfully connected to httpbin"
+    echo "Successfully connected to testbench"
   fi
 
   echo "${COLOR_GREEN}[ ======== ]${COLOR_RESET} Integration test environment set-up."
 }
 
+# Create most likely unique names for the project and bucket so multiple tests
+# can use the same testbench.
+readonly PROJECT_ID="fake-project-$(date +%s)"
+readonly BUCKET_NAME="fake-bucket-$(date +%s)"
+
 echo
 echo "Running Storage integration tests against local servers."
-start_httpbin
+start_testbench
 
+echo
 echo "Running storage::internal::CurlRequest integration test."
 ./curl_request_integration_test
 
+echo
+echo "Running storage::Bucket integration tests."
+./bucket_integration_test "${PROJECT_ID}" "${BUCKET_NAME}"
```

Comment:
Out of scope for this PR, but at some point we may want to allow running the integration tests against the actual service (as well as the emulator).  To that end, it would be nice to be able to plumb through the desired version and endpoint (via command line args?).  Just a thought to keep in mind as these are being written.

Reply:
Created #699 to track that.  Good idea too.